src/components/DateInput.jsx

```diff
@@ -90,7 +88,6 @@ class DateInput extends React.Component {
 
     if (focused && isFocused) {
       this.inputRef.focus();
-      this.inputRef.select();
```

Comment:
Can you elaborate in why the autoselection was removed?

Reply:
imo this is actually better for a11y. It's not expected that by default, moving to a text input will select everything in it.

Reply:
So before, when you focused on the input, it was select the whole text so that when you typed anything, it would just replace whatever was there. This seemed like a good approach when there was no cursor to indicate where you were in the input. However, with a cursor, it seems unnecessary behavior.